quiz-bundle-spec.md

```diff
@@ -32,6 +32,7 @@ default_locale     | ✓        | string     | Corresponds to the locale that th
 schema_version     | ✓        | integer    | Which version of the quiz bundle schema you are using
 title              |          | dictionary | A locale dictionary of the quiz title, such as "My Awesome Quiz"
 passing_percentage | ✓        | integer    | The threshold grade that a student needs to achieve in order to count as "passing" the quiz.
+fixed_place        |          | boolean    | `true` if this option should be fixed in place when shuffling the items (this is treated as `false` if it's missing).
```

Comment:
`true` if the items should be presented in a fixed order rather than shuffled (this is treated as `false` if it's missing).